i18n/fr.json

```diff
@@ -715,20 +715,20 @@
   "admin.elasticsearch.usernameDescription": "(Facultatif) Le nom d'utilisateur permettant de s'authentifier au serveur Elasticsearch.",
   "admin.elasticsearch.usernameExample": "Ex. : « elastic »",
   "admin.elasticsearch.usernameTitle": "Nom d'utilisateur du serveur :",
-  "admin.email.agreeHPNS": " Je comprends et j'accepte les [Conditions de Service](!https://about.mattermost.com/hpns-terms/) et de la [Politique de Confidentialité](!https://about.mattermost.com/hpns-privacy/) du service hébergé de notifications push de Mattermost.",
+  "admin.email.agreeHPNS": " Je comprends et j'accepte les [Conditions de Service](!https://mattermost.com/hpns-terms/) et de la [Politique de Confidentialité](!https://mattermost.com/data-processing-addendum/) du service hébergé de notifications push de Mattermost.",
   "admin.email.allowEmailSignInDescription": "Si activé, les utilisateurs peuvent se connecter avec leur adresse e-mail et leur mot de passe.",
   "admin.email.allowEmailSignInTitle": "Activer la connexion avec une adresse e-mail : ",
   "admin.email.allowSignupDescription": "Si activé, Mattermost autorise la création de comptes en utilisant une adresse e-mail et un mot de passe. Cette valeur devrait être désactivée seulement si vous souhaitez limiter l'authentification à un service d'authentification unique (SSO) comme AD/LDAP, SAML ou GitLab.",
   "admin.email.allowSignupTitle": "Activer la création de comptes avec une adresse e-mail : ",
   "admin.email.allowUsernameSignInDescription": "Si activé, les utilisateurs avec une authentification par e-mail peuvent se connecter en utilisant leur nom d'utilisateur et mot de passe. Ce paramètre n'affecte pas les connexions via AD/LDAP.",
   "admin.email.allowUsernameSignInTitle": "Activer la connexion avec nom d'utilisateur : ",
-  "admin.email.easHelp": "Pour en savoir plus sur la compilation et le déploiement de vos propres applications mobiles, la documentation relative à l'[App Store Entreprise](!https://about.mattermost.com/default-enterprise-app-store).",
+  "admin.email.easHelp": "Pour en savoir plus sur la compilation et le déploiement de vos propres applications mobiles, la documentation relative à l'[App Store Entreprise](!https://docs.mattermost.com/).",
```

Comment:
@cwarnermm I noticed that a few of these links now seem less-specific than the originals. I'm not sure if that was intentional or something that got missed in review for the previous PR